ffv1.md

```diff
@@ -94,10 +94,17 @@ Note: the operators and the order of precedence are the same as used in the C pr
 
 ### Mathematical functions
 
---------------------- -----------------------------------------------
-$$\lfloor a \rfloor$$ the largest integer less than or equal to a
-$$\lceil a \rceil$$   the smallest integer greater than or equal to a
---------------------- -----------------------------------------------
+--------------------- -------------------------------------------------
+$$\lfloor a \rfloor$$ the largest integer less than or equal to a.
+
+$$\lceil a \rceil$$   the smallest integer greater than or equal to a.
+
+abs(a)                The absolute value of a, i.e. abs(a) = sign(a)*a.
+
+log2(a)               The base-two logarithm of a.
+
+min(a,b)              The smallest of two values a and b.
+--------------------- -------------------------------------------------
```

Comment:
Sorry, again typography: I suggest either to start with a capital and end with a period, or to start with a minuscule and no period at the end. I prefer the second, because these are not plain sentences.